resources/qml/MessageInput.qml

```diff
@@ -195,7 +195,11 @@ Rectangle {
                     } else if (event.key == Qt.Key_Tab) {
                         event.accepted = true;
                         if (popup.opened) {
-                            popup.up();
+                            if (event.key == Qt.Key_Tab && (event.modifiers & Qt.ShiftModifier)) {
```

Comment:
oops I just noticed this `event.key == Qt.Key_Tab` part is not needed (already checked above), I'll rectify this